tests/HTTP/SapiTest.php

```diff
@@ -117,4 +117,49 @@ function testSend() {
 
     }
 
+    /**
+     * @runInSeparateProcess
+     */
+    function testSendLimitedByContentLengthString() {
+
+        $response = new Response(200);
+
+        $response->addHeader('Content-Length', 19);
+        $response->setBody('Send this sentence. Ignore this one.');
+
+        ob_start();
+
+        Sapi::sendResponse($response);
+
+        $result = ob_get_clean();
+        header_remove();
```

Comment:
I wonder if it could lead to errors in the test engine. Isn't it better to specify the header to remove instead?